vite.config.js

```diff
@@ -86,10 +86,17 @@ const getConfig = () => {
           target: `${URLStartPrefix}sso.azion.com`,
           cookieDomainRewrite: { '*': '' }
         }),
+        '^/api/v4/edge_sql': createProxyConfig({
+          target: `${URLStartPrefix}api.azion.com`,
+          rewrite: (path) => path.replace(/^\/api/, '')
+        }),
         '/api': createProxyConfig({
           target: `${URLStartPrefix}api.azion.com`,
           rewrite: (path) => path.replace(/^\/api/, '')
         }),
+        '^/v4/edge_sql': createProxyConfig({
+          target: `${URLStartPrefix}api.azion.com`
+        }),
```

Comment:
```suggestion
        '/api': createProxyConfig({
          target: `${URLStartPrefix}api.azion.com`,
          rewrite: (path) => path.replace(/^\/api/, '')
        }),
```
Utilize o /api ja existente